sql-cli/sql_cli/__main__.py

```diff
@@ -91,18 +91,12 @@ def validate(
     """
 )
 def run(
-    directory: Path = typer.Argument(
+    workflow_directory: Path = typer.Argument(
```

Comment:
Same here. I'd expect this to be `workflow_name` - and the name would match the directory of interest.